_assignments/grad_assignment.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: assignment
+title: "4 Credit Hour Assignment"
+parts:
+  - title: Proposal
+    due: Mar. 25, 2019 @ 11:59 PM
+  - title: Results
+    due: Apr. 22, 2019 @ 11:59 PM
+  - title: Final Submission
+    due: May 1, 2019 @ 11:59 PM
+material: false
+notes: false
+instructions: false
+rubric: false
+---
+
+Each individual taking the class for 4 credit hours will complete a small VR research project. Your task is to develop an idea for something that you can test in virtual reality, create a program or project add-on to test it with, run several people through your study and write up the results. You will first submit a research project proposal that will include a summary of previous scientific literature on your topic of interest, hypotheses (research questions that you will answer with your study) and a questionnaire to test your hypotheses that you will administer to subjects that participate in your study. After you have run your subjects, you will turn in copies of the questionnaires that you gave to your subjects and a short summary of your results. At the end of the semester, you will submit a short paper summarizing the findings from your mini research project. 
+
+- Read a few papers on a topic -- present at least 3-5 papers on a topic to investigate in a written paper the style of a lit review.
+  - Doesn’t have to be connected to your final project! 
```

Comment:
Not sure if this bullet is necessary - we don't mention that it can be connected to final until the very end.

Reply:
Deleted